src/Products/Jobber/log.py

```diff
@@ -36,6 +36,7 @@
     LoggingProxy,
     TaskLogFileHandler,
 )
+from ..Zuul.utils import try_unbrain
```

Comment:
So, it doesn't look like `try_unbrain` is used anywhere in this module.  Please remove unused imports.